iOS/samples/quickstart-swift/quickstart-swift/LaunchViewController.swift

```diff
@@ -0,0 +1,165 @@
+        bodyText.lineBreakMode = .byWordWrapping
+        bodyText.numberOfLines = 0
+        let screenSize = self.view.frame.height
+        if screenSize <= 667 {
+            // Font size for smaller iPhones.
+            bodyText.font = bodyText.font.withSize(14)
+            
+         } else if screenSize <= 812 {
+            // Font size for medium iPhones.
+            bodyText.font = bodyText.font.withSize(15)
+
+         } else if screenSize <= 896 {
+            // Font size for larger iPhones.
+            bodyText.font = bodyText.font.withSize(17)
+
+         } else if screenSize <= 1024 {
+            // Font size for iPads.
+            bodyText.font = bodyText.font.withSize(25)
+        } else {
+            // Font size for large iPads.
+            bodyText.font = bodyText.font.withSize(28)
+        }
+        view.addSubview(bodyText)
+        
+        launchButton = UIButton()
+        launchButton.backgroundColor = .darkGray
+        launchButton.contentEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        launchButton.setTitleColor(.white, for: .normal)
+        launchButton.setTitle("Immersive Reader", for: .normal)
+        launchButton.addTarget(self, action: #selector(launchImmersiveReaderButton(sender:)), for: .touchUpInside)
+        view.addSubview(launchButton)
+        
+        let layoutGuide = view.safeAreaLayoutGuide
+        
+        titleText.translatesAutoresizingMaskIntoConstraints = false
+        titleText.topAnchor.constraint(equalTo: layoutGuide.topAnchor, constant: 20).isActive = true
+        titleText.leadingAnchor.constraint(equalTo: layoutGuide.leadingAnchor, constant: 20).isActive = true
+        titleText.trailingAnchor.constraint(equalTo: layoutGuide.trailingAnchor, constant: -20).isActive = true
+        
+        bodyText.translatesAutoresizingMaskIntoConstraints = false
+        bodyText.topAnchor.constraint(equalTo: titleText.bottomAnchor, constant: 15).isActive = true
+        bodyText.leadingAnchor.constraint(equalTo: layoutGuide.leadingAnchor, constant: 20).isActive = true
+        bodyText.trailingAnchor.constraint(equalTo: layoutGuide.trailingAnchor, constant: -20).isActive = true
+        
+        launchButton.translatesAutoresizingMaskIntoConstraints = false
+        launchButton.widthAnchor.constraint(equalToConstant: 200).isActive = true
+        launchButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
+        launchButton.centerXAnchor.constraint(equalTo: layoutGuide.centerXAnchor).isActive = true
+        launchButton.bottomAnchor.constraint(equalTo: layoutGuide.bottomAnchor, constant: -10).isActive = true
+        
+        // Create content and options.
+        sampleChunk = Chunk(content: bodyText.text!, lang: nil, mimeType: nil)
+        sampleContent = Content(title: titleText.text!, chunks: [sampleChunk])
+        sampleOptions = Options(uiLang: nil, timeout: nil, uiZIndex: nil)
+        
```

Comment:
nit: space

Reply:
What do I need to change with the spacing?